packages/sources/superstate/src/transport/nav.ts

```diff
@@ -0,0 +1,202 @@
+import { Transport, TransportDependencies } from '@chainlink/external-adapter-framework/transports'
+import { BaseEndpointTypes, inputParameters } from '../endpoint/nav'
+import { isWeekend } from 'date-fns'
+import {
+  AdapterRequest,
+  AdapterResponse,
+  makeLogger,
+} from '@chainlink/external-adapter-framework/util'
+import { ResponseCache } from '@chainlink/external-adapter-framework/cache/response'
+import { Requester } from '@chainlink/external-adapter-framework/util/requester'
+import { calculateHttpRequestKey } from '@chainlink/external-adapter-framework/cache'
+import schedule from 'node-schedule'
+import {
+  getPreviousNonWeekendDay,
+  getStartingAndEndingDates,
+  isBeforeTime,
+  isInTimeRange,
+  toTimezoneDate,
+} from './utils'
+
+const logger = makeLogger('Superstate')
+
+export interface ResponseSchema {
+  fund_id: number
+  net_asset_value_date: string
+  net_asset_value: string
+  assets_under_management: string
+  outstanding_shares: string
+  net_income_expenses: string
+}
+
+const RETRY_INTERVAL_MINUTES = 1
```

Comment:
Minor: Might be worth making this configurable by env var, to give us a mitigation if we run into rate limiting